core/src/main/java/org/apache/accumulo/core/clientImpl/TabletServerBatchWriter.java

```diff
@@ -672,11 +672,11 @@ private class MutationWriter {
     public MutationWriter(int numSendThreads) {
       serversMutations = new HashMap<>();
       queued = new HashSet<>();
-      sendThreadPool = context.threadPools().getPoolBuilder(this.getClass().getName())
+      sendThreadPool = context.threadPools().getPoolBuilder("tserver.batch.writer.send.pool")
           .numCoreThreads(numSendThreads).build();
       locators = new HashMap<>();
-      binningThreadPool = context.threadPools().getPoolBuilder("BinMutations").numCoreThreads(1)
-          .withQueue(new SynchronousQueue<>()).build();
+      binningThreadPool = context.threadPools().getPoolBuilder("tserver.batch.writer.bin.mutations")
```

Comment:
`tserver` might be confusing, could probably just get away with `batch.writer...`

Reply:
removed in 3e2982c36b